src/Evolution/Systems/Ccz4/Tags.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/Ccz4/TagsDeclarations.hpp"
+#include "Evolution/Tags.hpp"
+#include "Options/Options.hpp"
+
+class DataVector;
+
+namespace Ccz4 {
+namespace Tags {
+/*!
+ * \brief Auxiliary variable which is analytically the spatial derivative of the
+ * natural log of the lapse
+ *
+ * \details If \f$ \alpha \f$ is the lapse, then we define
+ * \f$ A_i = \partial_i ln(\alpha) = \frac{\partial_i \alpha}{\alpha} \f$.
+ */
+template <size_t Dim, typename Frame>
+struct A : db::SimpleTag {
```

Comment:
Single-letter names can be problematic; they can be hard to search for and are very non-descriptive. How about more descriptive names like `DerivLogLapse`? What are your thoughts on this @nilsdeppe and @kidder?